common/text.c

```diff
@@ -1,22 +1,32 @@
 #include "text.h"
 
-static int s_textLang = /*CFG_LANGUAGE_EN*/1;
+static int s_textLang = SetLanguage_ENUS;
 
 void textInit(void) {   
     u64 LanguageCode=0;
     s32 Language=0;
-    setInitialize();
-    setGetSystemLanguage(&LanguageCode);
-    setMakeLanguage(LanguageCode, &Language);
-    s_textLang = Language;
-}
+    Result rc = setInitialize();
+    if (R_FAILED(rc)) s_textLang = SetLanguage_ENUS;
```

Comment:
Just init s_textLang here without R_FAILED(rc), and remove the duplicate lines below.